### DISCUSSION TITLE
Do I need to place kfmon.png in the root?

### DISCUSSION THREAD

**Discussion by kyxap** [koreader/koreader#12077] [Q&A] [answered]
Or I can place it inside of `.add` ?

Seems to work fine after restart but better to double check.

Main issue is to avoid getting it to calibre/kobo db/metadata

**Comment by pazos** [comment, marked as answer]
https://www.mobileread.com/forums/showpost.php?p=4019976&postcount=3

But, if you're fine with NickelMenu, you can remove those icons (or kfmon entirely)
